.github/workflows/install_ci.yml

```diff
@@ -24,13 +24,10 @@ jobs:
         with:
           check-latest: true
 
-      - name: Install via webscript (Windows)
-        if: runner.os == 'Windows'
-        run: curl -o install.vsh https://raw.githubusercontent.com/v-analyzer/v-analyzer/main/install.vsh; v run install.vsh; del install.vsh
-
-      - name: Install via webscript (nix)
-        if: runner.os != 'Windows'
-        run: v -e "$(curl https://raw.githubusercontent.com/v-analyzer/v-analyzer/main/install.vsh -fsSL)"
+      - name: Install via webscript
+        run: |
+          curl -o install.vsh https://raw.githubusercontent.com/v-analyzer/v-analyzer/main/install.vsh
+          v run install.vsh
```

Comment:
No, the idea here is to test directly what people that followed the README will do.
It is one line there, it should be 1 line here as well, even if it is long.